osc-server/src/test/java/org/osc/core/broker/service/request/DeleteDistributedApplianceRequestValidatorTest.java

```diff
@@ -150,6 +165,63 @@ public void testValidateAndLoad_WhenChainedToSfcDeleteRequest_ThrowsValidationEx
         this.validator.validateAndLoad(createRequest(VALID_ID_WITH_SFC, false));
     }
 
+    @Test
+    public void testValidateSfcToSecurityGroup_WhenDADeleteRequest_ThrowsValidationException() throws Exception {
```

Comment:
Please review the test naming convetion in the unit test guidelines. https://github.com/opensecuritycontroller/community/blob/master/development/code-guidelines/unit_test_guidelines.md#test-method-name

Reply:
@bmmukund it seems you missed this comment.